src/Pages/Details.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import { useLocation } from 'react-router';
+import './Details.css';
+
+const Details = () => {
+  const location = useLocation();
+  const {
+    // eslint-disable-next-line camelcase, max-len
```

Comment:
- [ ] [REQUIRED] Good job so far! Please fix linter errors without this kind of comment `/* eslint-disable ... */` because this type of comment is used in very few cases, and in your case, you can solve such errors yourself, and I think if implement my suggestion that this will increase your work experience and skills 😃